deploy/scripts/src/cri-orange/kbv.ts

```diff
@@ -62,26 +64,32 @@ export function kbv(): void {
     kbvAns3: string
   }
   const kbvAnsJSON: KbvAnswers = JSON.parse(kbvAnswersOBJ.kbvAnswers)
-  iterationsStarted.add(1)
+  iterationsStarted.add(3)
```

Comment:
```suggestion
  iterationsStarted.add(1)
```
I think some of the values have been changed where they weren't intended to be